packages/vite/src/node/plugin.ts

```diff
@@ -323,6 +323,20 @@ export interface Plugin<A = any> extends RollupPlugin<A> {
   >
 }
 
+export interface CustomPluginOptionsVite {
+  /**
+   * If this is a CSS Rollup module, you can scope to its importer's exports
+   * so that if those exports are treeshaken away, the CSS module will also
+   * be treeshaken.
+   *
+   * Example config if the CSS id is `/src/App.vue?vue&type=style&lang.css`:
+   * ```js
+   * cssScopeTo: ['/src/App.vue', 'default']
+   * ```
+   */
+  cssScopeTo?: [string, string | undefined]
```

Comment:
I wonder if naming this `cssAttachTo` is better. I'm not sure why but calling this `scopedTo` doesn't sound a bit off to me.

Reply:
I fine with any name. I was thinking of "scope" because frameworks call them scoped css, and extrapolating that to "what it's scope to?" 😄

Reply:
After thinking back and forth, I ended up with scopedTo 😄